app/Models/Extensions/Thumb.php

```diff
@@ -81,11 +86,16 @@ public static function createFromQueryable(Relation|Builder $photoQueryable, Sor
 	 */
 	public static function createFromPhoto(?Photo $photo): ?Thumb
 	{
-		$thumb = $photo?->size_variants->getThumb();
+		$thumb = config('app.livewire', true) === true
+			? $photo?->size_variants->getSmall() ?? $photo?->size_variants->getThumb()
+			: $photo?->size_variants->getThumb();
```

Comment:
Oh gosh, a nested ternary, with no brackets no less! :scream: Would the following achieve the same effect?

```suggestion
		$thumb = config('app.livewire', true) === true && $photo?->size_variants->getSmall()
		    ? $photo?->size_variants->getSmall()
		    : $photo?->size_variants->getThumb();
```

Reply:
Fixed [736d5ee](https://github.com/LycheeOrg/Lychee/pull/2138/commits/736d5ee064ca089c9b8cb3d05de30f8d8dc563be)